one/src/events.rs

```diff
@@ -1,17 +1,21 @@
+use anyhow::Result;
 use ceramic_p2p::SQLiteBlockStore;
 use chrono::{SecondsFormat, Utc};
 use cid::{multibase, multihash, Cid};
-use clap::Parser;
-use clap_verbosity_flag::{InfoLevel, Verbosity};
+use clap::{Args, Subcommand};
 use clio::{ClioPath, InputPath, OutputPath};
 use glob::{glob, Paths};
 use sqlx::sqlite::SqlitePool;
 use std::{fs, path::PathBuf};
 
-/// Migrate from kubo to ceramic-one
-#[derive(Parser, Debug)]
-#[clap(name = "migration", about, author, version, about)]
-struct Cli {
+#[derive(Subcommand, Debug)]
+pub enum EventsCommand {
```

Comment:
This EventsCommand will hold the Event processing tasks like making eventIDs for recon for blocks in the block store.